scripts/versioner.py

```diff
@@ -227,6 +223,10 @@ def get_app():
         git_root = subprocess.check_output(root_command, encoding='utf-8').rstrip()
         app_name = os.path.basename(git_root).rstrip().lower()
 
+        # If we're nested within MOOSE_DIR, we're moose_combined app
+        if app_name == 'moose':
```

Comment:
The more I think about this, this'll fail if someone calls their directory not "moose" or "moose.git". Do we care?

Reply:
we can strip off the .git if its there, but yeah I am not sure what to do if the user decides to rename the moose dir to something else entirely...